lib/rsgem/tasks/create_gem.rb

```diff
@@ -32,6 +33,23 @@ def shell_command
           bundler_options
         ].compact.join(' ')
       end
+
+      def ensure_author
+        temp_username = 'change_me'
+        temp_email = 'change_me@notanemail.com'
+        gemspec = File.read(context.gemspec_path)
+        unless system('git config user.email')
```

Comment:
I think this will always be evalueated as truthy, won't it?

Reply:
I tested this with pry and then with the unit test to make sure it only overwrote the values if necessary. I can triple check though to make sure the return value of `system('git config user.email')` when its blank is `false`.

Reply:
I did test the same thing, with IRB instead of pry and I always got `true` surprisingly.

Reply:
https://rubyapi.org/2.7/o/kernel#method-i-system so according to this

>system returns true if the command gives zero exit status, false for non zero exit status. Returns nil if command execution fails. An error status is available in $?.

Reply:
Ok I must have a bug in the test then and its always truthy. I'll remedy this.

Reply:
Since the tests use backticks and analyze the output, I did some manual testing in my CLI and here are the results:

1. Unset my git email 
2. Run `rsgem new foo` and inspect results => value is `false`
3. Set my git email
4. Run `rsgem new foo` and inspect results => value is `true`

So it looks like this works as expected. I will still move it to a new task. 

```sh
20:11:02 | rsgem | (error-if-no-github-user) -> git config --unset user.email

37: def ensure_author
    38:   temp_username = 'change_me'
    39:   temp_email = 'change_me@notanemail.com'
    40:   gemspec = File.read(context.gemspec_path)
    41:   require 'pry'
 => 42:   binding.pry
    43:   unless system('git config user.email')
    44:     puts "Warning: No git email set, setting #{temp_email} for now, please change this before publishing your gem."
    45:     gemspec.gsub!(/(spec.email[^\[]+)\[([^\]]+)/, "\\1\['#{temp_email}'")
    46:   end
    47:
    48:   unless system('git config user.name')
    49:     puts "Warning: No git username set, setting #{temp_username} for now, please change this before publishing your gem."
    50:     gemspec.gsub!(/(spec.authors[^\[]+)\[([^\]]+)/, "\\1\['#{temp_username}'")
    51:   end
    52:
    53:   File.write(context.gemspec_path, gemspec)
    54: end

[1] pry(#<RSGem::Tasks::CreateGem>)> value = system('git config user.email')
=> false

----------

20:13:03 | rsgem | (error-if-no-github-user) -> git config user.email 'yesbeckjs@gmail.com'
    37: def ensure_author
    38:   temp_username = 'change_me'
    39:   temp_email = 'change_me@notanemail.com'
    40:   gemspec = File.read(context.gemspec_path)
    41:   require 'pry'
 => 42:   binding.pry
    43:   unless system('git config user.email')
    44:     puts "Warning: No git email set, setting #{temp_email} for now, please change this before publishing your gem."
    45:     gemspec.gsub!(/(spec.email[^\[]+)\[([^\]]+)/, "\\1\['#{temp_email}'")
    46:   end
    47:
    48:   unless system('git config user.name')
    49:     puts "Warning: No git username set, setting #{temp_username} for now, please change this before publishing your gem."
    50:     gemspec.gsub!(/(spec.authors[^\[]+)\[([^\]]+)/, "\\1\['#{temp_username}'")
    51:   end
    52:
    53:   File.write(context.gemspec_path, gemspec)
    54: end

[1] pry(#<RSGem::Tasks::CreateGem>)> value = system('git config user.email')
yesbeckjs@gmail.com
```